regression/ansi-c/goto_convert/switch_range_empty/test.desc

```diff
@@ -0,0 +1,8 @@
+CORE
+main.c
+
+^EXIT=134$
```

Comment:
Huh? That's not what should be happening...

Reply:
What should be happening? I see that `gcc` and `clang` produce warnings that the range is empty, but are happy to compile nonetheless. Should I remove the invariant?

Reply:
We should be able to support whatever GCC and Clang do. 134 (a failing invariant) is not an exit code we should ever be producing for a syntactically valid C program.